.github/workflows/e2e-test.yml

```diff
@@ -0,0 +1,62 @@
+name: E2E Test
+
+on:
+  pull_request:
+    branches:
+    - 'main'
+    - 'release-*'
+    paths-ignore:
+    - '**/*.md'
+    - 'docs/**'
+    - '.gitignore'
+    - 'hack/*.sh'
+    - 'LICENSE'
+    - 'SECURITY_CONTACTS'
+    - 'DCO'
+    - 'OWNERS'
+
+permissions: {}
+
+jobs:
+  test:
+    name: e2e test
+    runs-on: ubuntu-latest-8-cores
+    strategy:
+      fail-fast: false
+      matrix:
+        bmc-protocol:
+        # - redfish
+        - redfish-virtualmedia
+        # - ipmi
```

Comment:
shall we comment out ipmi and test how is it behaving ?

Reply:
In the current tests we run both redfish-virtualmedia and ipmi on PRs and all on periodics. I commented out all but one here to conserve resources while testing. I'll push an update with periodic job and the same protocols tested as we have now :slightly_smiling_face: